force-app/main/default/classes/STG_PanelUserRollup_CTRL.cls

```diff
@@ -381,6 +381,19 @@ public with sharing class STG_PanelUserRollup_CTRL extends STG_Panel {
         ApexPages.addMessage(myMsg);
     }
 
+    public static void throwAuraExceptionIfRunningUserNotAdmin() {
```

Comment:
I like the method abstraction, however this particular class is for a legacy feature (not part of CRLP). If all (or at least almost all) NPSP Settings pages require the permissions as defined in `runningUserIsAdmin()`, couldn't this method just exist in the same `STG_Panel` class? I can easily see this being used by other Aura/LWC components that are referenced in NPSP Settings, such as Advanced Field Mapping, RD2 enablement, and RD2 status mapping. All would have at least this same perm requirement.

Reply:
I agree with this, since it'll probably be used in many of the classes that extends STG_Panel or are related to STG_Panel in some way, that's a decent spot for it.